Comparing source compatibility of C:\au\builds\uimaj-3.0.2\rc4\artifacts\target\checkout\uimaj-core\target\uima-core.jar against C:\Users\MarshallSchor\.m2\repository\org\apache\uima\uimaj-core\3.0.1\uimaj-core-3.0.1.jar
===! UNCHANGED CLASS: PUBLIC org.apache.uima.cas.admin.CASAdminException  (compatible)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
***! MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.uima.cas.ArrayFS  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	***! MODIFIED METHOD: PUBLIC ABSTRACT org.apache.uima.cas.FeatureStructure (<-org.apache.uima.jcas.cas.TOP) get(int)
		===  UNCHANGED EXCEPTION: java.lang.ArrayIndexOutOfBoundsException
===! UNCHANGED CLASS: PUBLIC org.apache.uima.cas.CASRuntimeException  (compatible)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
***! MODIFIED CLASS: PUBLIC org.apache.uima.jcas.cas.AnnotationBase  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	***! MODIFIED METHOD: PUBLIC org.apache.uima.cas.SofaFS (<-org.apache.uima.jcas.cas.Sofa) getSofa()
***! MODIFIED CLASS: PUBLIC FINAL org.apache.uima.jcas.cas.FSArray  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===! UNCHANGED INTERFACE: org.apache.uima.cas.ArrayFS
	***! MODIFIED METHOD: PUBLIC org.apache.uima.cas.FeatureStructure (<-org.apache.uima.jcas.cas.TOP) get(int)
***! MODIFIED CLASS: PUBLIC org.apache.uima.jcas.cas.NonEmptyFloatList  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	---! REMOVED CONSTRUCTOR: PUBLIC(-) NonEmptyFloatList(org.apache.uima.jcas.JCas, float, org.apache.uima.jcas.cas.CommonList)
	+++  NEW CONSTRUCTOR: PUBLIC(+) NonEmptyFloatList(org.apache.uima.jcas.JCas, float, org.apache.uima.jcas.cas.FloatList)
***! MODIFIED CLASS: PUBLIC org.apache.uima.jcas.cas.NonEmptyFSList  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	---! REMOVED CONSTRUCTOR: PUBLIC(-) NonEmptyFSList(org.apache.uima.jcas.JCas, org.apache.uima.jcas.cas.TOP, org.apache.uima.jcas.cas.CommonList)
	+++  NEW CONSTRUCTOR: PUBLIC(+) NonEmptyFSList(org.apache.uima.jcas.JCas, org.apache.uima.jcas.cas.TOP, org.apache.uima.jcas.cas.FSList)
***! MODIFIED CLASS: PUBLIC org.apache.uima.jcas.cas.NonEmptyIntegerList  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	---! REMOVED CONSTRUCTOR: PUBLIC(-) NonEmptyIntegerList(org.apache.uima.jcas.JCas, int, org.apache.uima.jcas.cas.CommonList)
	+++  NEW CONSTRUCTOR: PUBLIC(+) NonEmptyIntegerList(org.apache.uima.jcas.JCas, int, org.apache.uima.jcas.cas.IntegerList)
***! MODIFIED CLASS: PUBLIC org.apache.uima.jcas.cas.NonEmptyStringList  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	---! REMOVED CONSTRUCTOR: PUBLIC(-) NonEmptyStringList(org.apache.uima.jcas.JCas, java.lang.String, org.apache.uima.jcas.cas.CommonList)
	+++  NEW CONSTRUCTOR: PUBLIC(+) NonEmptyStringList(org.apache.uima.jcas.JCas, java.lang.String, org.apache.uima.jcas.cas.StringList)
===  UNCHANGED CLASS: PUBLIC org.apache.uima.pear.util.FileUtil  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED METHOD: PUBLIC STATIC boolean copyFile(java.io.File, java.io.File)
		+++  NEW ANNOTATION: java.lang.Deprecated
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC STATIC boolean copyFile(java.net.URL, java.io.File)
		+++  NEW ANNOTATION: java.lang.Deprecated
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC STATIC java.io.File createTempFile(java.lang.String, java.lang.String)
		+++  NEW ANNOTATION: java.lang.Deprecated
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC STATIC long getFileSize(java.lang.String)
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String[] loadListOfStrings(java.io.File)
		+++  NEW ANNOTATION: java.lang.Deprecated
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String loadTextFile(java.io.File)
		+++  NEW ANNOTATION: java.lang.Deprecated
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC STATIC java.lang.String loadTextFile(java.io.File, java.lang.String)
		+++  NEW ANNOTATION: java.lang.Deprecated
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC STATIC boolean moveFile(java.io.File, java.io.File)
		+++  NEW ANNOTATION: java.lang.Deprecated
		===  UNCHANGED EXCEPTION: java.io.IOException
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.uima.resource.ResourceManagerPearWrapper  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	+++  NEW ANNOTATION: java.lang.Deprecated
===! UNCHANGED CLASS: PUBLIC org.apache.uima.UIMA_IllegalArgumentException  (compatible)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
===! UNCHANGED CLASS: PUBLIC org.apache.uima.UIMA_IllegalStateException  (compatible)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
===! UNCHANGED CLASS: PUBLIC org.apache.uima.UIMA_UnsupportedOperationException  (compatible)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
***! MODIFIED CLASS: PUBLIC org.apache.uima.UIMARuntimeException  (field removed)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	---! REMOVED FIELD: PUBLIC(-) STATIC(-) FINAL(-) java.lang.String resource_file
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String STANDARD_MESSAGE_CATALOG
===  UNCHANGED CLASS: PUBLIC org.apache.uima.util.FileUtils  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED METHOD: PUBLIC STATIC FINAL void copyFile(java.io.File, java.io.File)
		+++  NEW ANNOTATION: java.lang.Deprecated
		===  UNCHANGED EXCEPTION: java.io.IOException
	===  UNCHANGED METHOD: PUBLIC STATIC FINAL java.io.File createTempDir(java.io.File, java.lang.String)
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED METHOD: PUBLIC STATIC FINAL java.io.File createTempFile(java.lang.String, java.lang.String, java.io.File)
		+++  NEW ANNOTATION: java.lang.Deprecated
		===  UNCHANGED EXCEPTION: java.io.IOException
